charts/telemetry-collector/templates/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+{{- include
+      "accelleran.common.deployment"
+      (fromYaml (include "accelleran.telemetry-collector.deployment.args" $))
+-}}
+
+{{- define "accelleran.telemetry-collector.deployment.args" -}}
+{{- $ := . -}}
+{{- $fullname := include "accelleran.common.fullname" (dict "top" $) -}}
+
+top:
+  {{ $ | toYaml | nindent 2 }}
+
+volumes:
+- name: {{ $fullname }}-config-volume
+  emptyDir: {}
+- name: {{ $fullname }}-config
+  configMap:
+    name: {{ $fullname }}-config
+
+
+volumeMounts:
+- name: {{ $fullname }}-config
+  mountPath: /app/config.yaml
+  subPath: "config.yaml"
+- name: {{ $fullname }}-config-volume
+  mountPath: /config
```

Comment:
Not sure if I understood this properly, but the application didn't write anything to the file system I thought? So I think we can remove this

Reply:
Yes correct - done